test/extended/apiserver/webhook.go

```diff
@@ -0,0 +1,292 @@
+package apiserver
```

Comment:
could we move this to openshift-apiserver repo?

Reply:
yes, we could. We would have to move the common code to `library-go` repo.